src/cli/export/export.go

```diff
@@ -37,3 +55,119 @@ func exportCmd() *cobra.Command {
 func handleExportCmd(cmd *cobra.Command, args []string) error {
 	return cmd.Help()
 }
+
+func runExport(
+	ctx context.Context,
+	cmd *cobra.Command,
+	args []string,
+	ueco utils.ExportCfgOpts,
+	sel selectors.Selector,
+	backupID, serviceName string,
+) error {
+	r, _, _, _, err := utils.GetAccountAndConnect(ctx, sel.PathService(), repo.S3Overrides(cmd))
+	if err != nil {
+		return Only(ctx, err)
+	}
+
+	defer utils.CloseRepo(ctx, r)
+
+	exportLocation := args[0]
+	if exportLocation == "" {
+		// This should not be possible, but adding it just in case.
+		exportLocation = control.DefaultRestoreLocation + dttm.FormatNow(dttm.HumanReadableDriveItem)
+	}
+
+	Infof(ctx, "Exporting to folder %s", exportLocation)
+
+	eo, err := r.NewExport(
+		ctx,
+		backupID,
+		sel,
+		utils.MakeExportConfig(ctx, ueco),
+	)
+	if err != nil {
+		return Only(ctx, clues.Wrap(err, "Failed to initialize "+serviceName+" export"))
+	}
+
+	expColl, err := eo.Run(ctx)
+	if err != nil {
+		if errors.Is(err, data.ErrNotFound) {
+			return Only(ctx, clues.New("Backup or backup details missing for id "+backupID))
+		}
+
+		return Only(ctx, clues.Wrap(err, "Failed to run "+serviceName+" export"))
+	}
+
+	// It would be better to give a progressbar than a spinner, but we
+	// have know way of knowing how many files are available as of now.
+	diskWriteComplete := observe.MessageWithCompletion(ctx, "Writing data to disk")
+	defer func() {
+		diskWriteComplete <- struct{}{}
+		close(diskWriteComplete)
+	}()
+
+	err = writeExportCollections(ctx, exportLocation, expColl)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func writeExportCollections(
```

Comment:
The CLI layer is the wrong owner for this logic, as well as `writeExportItem`.  That should probably move into the `export` package.  That will ensure the functionality 1/ exists independently of the top level api layers,  2/ is accessible to SDK consumers, and 3/ will likely be easier to test in the long term.

Also, naming nit.  That will keep the func name in line with our other naming conventions.
```suggestion
func ConsumeExportCollections(
```